tensorflow_datasets/core/download/kaggle.py

```diff
@@ -168,7 +169,14 @@ def download_file(self, fname, output_dir):
     if self._kaggle_type.extra_flag:
       command.append(self._kaggle_type.extra_flag)
     _run_kaggle_command(command, self._competition_name)
-    return os.path.join(output_dir, fname)
+    # kaggle silently compresses some files to '.zip` files.
```

Comment:
Also we should add a comment to use `--unzip` once supported by kaggle (https://github.com/Kaggle/kaggle-api/issues/9)